pom.xml

```diff
@@ -152,6 +153,30 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>com.mycila.maven-license-plugin</groupId>
+                        <artifactId>maven-license-plugin</artifactId>
+                        <version>1.10.b1</version>
+                        <configuration>
+                            <header>${license.dir}/copyright_header.txt</header>
+                            <properties>
+                                <project>${project.name}</project>
+                                <founder>${project.organization.name}</founder>
+                                <year>${project.inceptionYear}</year>

```

Comment:
Do we have to set these properties? We're not using any placeholders in the template file.

Reply:
It is working as expected even without these  properties, i will be deleting these properties from pom.